tools/build_image.sh

```diff
@@ -127,47 +127,29 @@ env PKG_INSTALL_ROOT=$ROOT \
     SVCCFG_REPOSITORY=$SVCCFG_REPOSITORY \
     SVCCFG_CHECKHASH=1 $SVCCFG import \
 		       -p /dev/stdout $ROOT/lib/svc/manifest
+SVCCFG_REPOSITORY=$SVCCFG_REPOSITORY \
```

Comment:
I don't think this should be necessary, because cpio is the default, right?  I just hacked around that, rather than changing the real default (... I wonder why I did that?).

I tried removing this and things seemed fine, but I did much less testing than you did so I wanted to ask